### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by klawani**
### Change description ###
update readme

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -57,4 +57,17 @@
     <filePath regex="true">.*postgresql-1.11.4.jar</filePath>
     <cpe>cpe:/a:postgresql:postgresql</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+			Doesn't look exploitable, will wait for new jackson databind library
```
Comment (adusumillipraveen) [inline config/owasp/suppressions.xml]: There is already new jackson databind library.

Comment (klawani) [reply config/owasp/suppressions.xml]: What do you mean? This change seems to fix the issue

Comment (adusumillipraveen) [reply config/owasp/suppressions.xml]: I meant this : https://github.com/FasterXML/jackson-databind/releases/tag/jackson-databind-2.9.10

Comment (adusumillipraveen) [reply config/owasp/suppressions.xml]: I meant this : https://github.com/FasterXML/jackson-databind/releases/tag/jackson-databind-2.9.10

**Pull request closed without merging**